mmdeploy/codebase/mmdet/core/post_processing/bbox_nms.py

```diff
@@ -405,9 +407,14 @@ def multiclass_nms__torchscript(ctx,
 class AscendBatchNMSOp(torch.autograd.Function):
 
     @staticmethod
-    def forward(ctx, bboxes: torch.Tensor, scores: torch.Tensor,
-                score_threshold: float, iou_threshold: float,
-                max_size_per_class: int, max_total_size: int):
+    def forward(ctx,
+                bboxes: torch.Tensor,
+                scores: torch.Tensor,
+                score_threshold: float,
+                iou_threshold: float,
+                max_size_per_class: int,
+                max_total_size: int,
+                dummy_output: Any = None):
```

Comment:
What's the motivation for `dummy_output`?

Reply:
PyTorch op can share the same outputs with custom op.